src/schema/v2/home/home_page_artist_module.ts

```diff
@@ -56,7 +57,7 @@ export const HomePageArtistModuleTypes: {
   TRENDING: {
     description: "The trending artists.",
     display: () => Promise.resolve(true),
```

Comment:
One follow-up, after reviewing the Eigen side of this.

**thought:** My 2¢ is that we could consider adding a new key (a la `CURATED_TRENDING`) instead of redefining `TRENDING` here.

In my earlier PR work that I felt the "trending" nomenclature was sort of reserved for our algorithmic concept of trending artists (whether artsy/delta today or something else in the future) and therefore tried to distinguish this human-curated version with the naming of the field. 

Do with that what you will!